app/services/clients/products_client.js

```diff
@@ -91,9 +96,29 @@ function getProductsByGatewayAccountId (gatewayAccountId) {
   }).then(products => products.map(product => new Product(product)))
 }
 
+/**
+ * @param {String} productExternalId: the external id of the product you wish to retrieve
+ * @returns {Promise<Product>}
+ */
+function updateProductServiceName (productExternalId, serviceName) {
+  return baseClient.patch({
+    headers,
+    baseUrl,
+    url: `/products`,
```

Comment:
same as comments as in https://github.com/alphagov/pay-selfservice/pull/619#pullrequestreview-88451012